### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/android-apps/actions/runs/4094419408#summary-11114296906

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/android-apps/actions/runs/4094419544#summary-11114297087

### CODE REVIEW

File: dashlane-project/Dashlane/src/main/java/com/dashlane/db/CreationModificationDateTableWorker.java [hunk 1]
```diff
@@ -308,7 +308,7 @@ private static class PendingUpdateItem {
         private SyncObjectType mDataType;
         private String mCreationDate;
         private String mUserModificationDate;
-        private String mLocallyViewdDate;
+        private String mLocallyViewedDate;
```
Comment (jsoref) [inline dashlane-project/Dashlane/src/main/java/com/dashlane/db/CreationModificationDateTableWorker.java:311]: I'm not particularly confident about `Viewed` vs `View`

File: dashlane-project/Dashlane/src/main/java/com/dashlane/util/hardwaresecurity/AuthModuleLogger.kt [hunk 2]
```diff
@@ -29,7 +29,7 @@ class AuthModuleLogger(
     }
 
     fun logUsageLogoutFromBiometrics() {
-        sendUsageLog35("askLgout")
+        sendUsageLog35("askLogout")
```
Comment (jsoref) [inline dashlane-project/Dashlane/src/main/java/com/dashlane/util/hardwaresecurity/AuthModuleLogger.kt:32]: It took me a bit, but I've decided that this aren't actually tied to a real third party thing, but they'd probably break log counting. It shouldn't be terribly hard for a backend to merge the two names, and realistically it'll need to handle such things eventually.

That said, I'm happy to drop this or anything else.

File: dashlane-project/Dashlane/src/main/res/layout/item_section_header.xml [hunk 3]
```diff
@@ -10,5 +10,5 @@
     android:paddingBottom="11.5dp"
     android:textAppearance="?textAppearanceSubtitle1"
     android:textColor="@color/text_neutral_quiet"
-    tools:text="Passsword Manager"
+    tools:text="Password Manager"
     />
```
Comment (jsoref) [inline dashlane-project/Dashlane/src/main/res/layout/item_section_header.xml:14]: 1. I try not to fix the missing newline at eof warning even though I (like Git and GitHub) dislike it.
2. Hanging here, I'm renaming the following file. I think I've picked the correct name, but I'm not certain.

File: dashlane-project/Dashlane/src/main/res/values/string_autofill_onboarding.xml [hunk 4]
```diff
@@ -7,7 +7,7 @@
     <string name="onboarding_in_app_login_step2_positive_button">Go to Settings</string>
 
     <string name="in_app_login_persistent_notification_title">Dashlane In App Login is running</string>
-    <string name="in_app_login_persistent_notification_message">Dashlane may help you logging in to this app</string>
+    <string name="in_app_login_persistent_notification_message">Dashlane may help you logging into this app</string>
```
Comment (jsoref) [inline dashlane-project/Dashlane/src/main/res/values/string_autofill_onboarding.xml:10]: There are differences of opinion about whether `log in` is a magical verb that should be treated as separate from `to`. I personally believe that `log` is a verb and you can `log into` a thing. If your style owner objects, I'm happy to drop this.

File: dashlane-project/modules/features/authenticator/src/main/java/com/dashlane/authenticator/AuthenticatorMultipleMatchesResult.kt [hunk 5]
```diff
@@ -21,7 +21,7 @@ class AuthenticatorMultipleMatchesResult : DashlaneActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_mutliple_matches_result)
+        setContentView(R.layout.activity_multiple_matches_result)
         setSupportActionBar(findViewById(R.id.toolbar))
         supportActionBar?.apply {
             setDisplayHomeAsUpEnabled(true)
```
Comment (jsoref) [inline dashlane-project/modules/features/authenticator/src/main/java/com/dashlane/authenticator/AuthenticatorMultipleMatchesResult.kt:27]: Another renamed file follows

File: dashlane-project/modules/features/csv-import/src/main/java/com/dashlane/csvimport/ImportFromChromeLogger.kt [hunk 6]
```diff
@@ -64,7 +64,7 @@ class ImportFromChromeLogger(
 
     fun logCsvImportAddManuallyClicked() = log(
         subtype = "chrome_android_csv",
-        action = "click_add_manually_after_errror_message_onboarding_screen"
+        action = "click_add_manually_after_error_message_onboarding_screen"
     )
 
     private fun log(
```
Comment (jsoref) [inline dashlane-project/modules/features/csv-import/src/main/java/com/dashlane/csvimport/ImportFromChromeLogger.kt:70]: Renamed file follows

File: dashlane-project/modules/nfccardreader/src/main/java/com/github/devnied/emvnfccard/model/enums/CountryCodeEnum.java [hunk 7]
```diff
@@ -333,7 +333,7 @@ public enum CountryCodeEnum implements IKeyEnum {
 
 	
 
-	GG("Guemsey", "GGY", 831),
+	GG("Guernsey", "GGY", 831),
```
Comment (jsoref) [inline dashlane-project/modules/nfccardreader/src/main/java/com/github/devnied/emvnfccard/model/enums/CountryCodeEnum.java:336]: This is fairly notable.

I'm pretty sure it's a reference to https://en.wikipedia.org/wiki/Guernsey

**Comment by jsoref** [comment]
Fwiw, I'm pretty sure this includes the corrections from #1.

Please feel free to take the changes however, I'm not attached to anything in particular. Mostly, I like to use software and not wince when I see typos. (And Dashlane is definitely software I might use or see someone else using...)

**Comment by Mikescops** [comment]
Hello,

Thanks for your typo corrections 👍 indeed we'll merge with #1 

Sorry we don't accept contributions on Github for now because we don't have the right tooling to do so yet.
That said we can pickup your changes upstream and mention you as the author so it appears in the next release.

@xfalempin  does it seems ok to you?

**Pull request closed without merging**